<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="37394" started-at="2019-09-20T11:39:28Z" finished-at="2019-09-20T11:40:05Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="37394" started-at="2019-09-20T11:39:28Z" finished-at="2019-09-20T11:40:05Z">
      <class name="RunFile.ExecuteFile">
        <test-method status="PASS" signature="setup()[pri:0, instance:RunFile.ExecuteFile@475530b9]" name="setup" is-config="true" duration-ms="13619" started-at="2019-09-20T17:24:28Z" finished-at="2019-09-20T17:24:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="validSignup()[pri:0, instance:RunFile.ExecuteFile@475530b9]" name="validSignup" duration-ms="23039" started-at="2019-09-20T17:24:41Z" finished-at="2019-09-20T17:25:04Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.google.common.base.Preconditions.checkNotNull(Preconditions.java:787)
	at org.openqa.selenium.interactions.Actions.<init>(Actions.java:68)
	at commonLibs.JavaScripExecutor.hoverAndClick(JavaScripExecutor.java:62)
	at supportFile.BuyItem.addtocard(BuyItem.java:32)
	at RunFile.ExecuteFile.validSignup(ExecuteFile.java:19)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validSignup -->
        <test-method status="PASS" signature="close()[pri:0, instance:RunFile.ExecuteFile@475530b9]" name="close" is-config="true" duration-ms="720" started-at="2019-09-20T17:25:04Z" finished-at="2019-09-20T17:25:05Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
      </class> <!-- RunFile.ExecuteFile -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
